src/main/java/teammates/common/datatransfer/InstructorPrivileges.java

```diff
@@ -350,7 +375,34 @@ public boolean isAllowedForPrivilege(String sectionName, String sessionName, Str
     public boolean isAllowedForPrivilegeAnySection(String sessionName, String privilegeName) {
         return isAllowedInSessionLevelAnySection(sessionName, privilegeName);
     }
-    
+
+    public boolean hasCoownerPrivileges() {
+        return hasSamePrivileges(coownerPrivileges);
+    }
+
+    public boolean hasManagerPrivileges() {
+        return hasSamePrivileges(managerPrivileges);
+    }
+
+    public boolean hasObserverPrivileges() {
+        return hasSamePrivileges(observerPrivileges);
+    }
+
+    public boolean hasTutorPrivileges() {        
+        return hasSamePrivileges(tutorPrivileges);
+    }
+
+    private boolean hasSamePrivileges(HashMap<String, Boolean> defaultPrivileges) {
+
+        for (String privilege : defaultPrivileges.keySet()) {
```

Comment:
Will it be better if we have at least an enum for the permissions? Then at least we can iterate over the enum values instead of assuming that all permissions are represented in this hashmap